src/permutat.cc

```diff
@@ -1125,6 +1125,40 @@ UInt LargestMovedPointPerm(Obj perm)
         return LargestMovedPointPerm_<UInt4>(perm);
 }
 
+/****************************************************************************
+**
+*F  SmallestMovedPointPerm( <perm> ) smallest point moved by perm
+**
+**  'SmallestMovedPointPerm' returns  the  smallest  positive  integer that is
```

Comment:
But `Infinity` is not an integer. Shouldn't it say something like: `'SmallestMovedPointPerm' returns  the  smallest  positive  integer that is moved by the permutation <perm> or Infinity if <perm> is the identity.`

Reply:
Indeed. Could also take the text used in the manual for SmallestMovedPoint. Or, for that matter, just say
> This function implements the operation `SmallestMovedPoint` for internal permutations.

Also, don't use those double spaces, please :-)